docs/tutorials/projects/verification-system.md

```diff
@@ -0,0 +1,240 @@
+| `trigger_2_labels` | array | **Required** | The class names from `trigger_2_detector` that count as valid. |
+| `trigger_2_confidence` | float | Optional | The detection confidence needed in order to move into the `COUNTDOWN` state. <br> Default: `0.5` |
+| `verification_detector` | string | **Required** | The name of the vision service detector that you want to use to verify the object. |
+| `verification_labels` | array | **Required** | The class names from `verification_detector` that count as valid. |
+| `verification_confidence` | float | Optional | The detection confidence needed in order to move into the `DISARMED` state. <br> Default: `0.8` |
+| `countdown_time_s` | int | Optional | The time in seconds the system will remain in state `COUNTDOWN` before transitioning to state `ALARM`. <br> Default: `20` |
+| `alarm_time_s` | int | Optional | The time in seconds the system will remain in  state `ALARM` before transitioning to state `TRIGGER_1`. <br> Default: `10` |
+| `disarmed_time_s` | int | Optional | The time in seconds the system will remain in  state `DISARMED` before transitioning to state `TRIGGER_1`. <br> Default: `10` |
+| `disable_alarm` | bool | Optional | Disables the `COUNTDOWN` and `ALARM` states. The system will always remain the `TRIGGER_1` and `TRIGGER_2` states. <br> Default: `false` |
+
+## Configure a transform camera
+
+To view the classifications that your verification system makes, configure a transform camera.
+
+Navigate to the **Config** tab of your robot’s page in [the Viam app](https://app.viam.com/).
+Click on the **Components** subtab and click **Create component**. Select the `camera` type, then select the `transform` model. Enter a name for your camera and click **Create**.
+
+With **Builder** mode selected, copy and paste the following attributes JSON into your camera's **Attributes** box:
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "pipeline": [
+    {
+      "type": "classifications",
+      "attributes": {
+        "classifier_name": "security",
+        "confidence_threshold": 0.5
+      }
+    }
+  ],
+  "source": "my-webcam"
+}
+```
+
+Note that the camera name in `"source"` and the `"classifier_name"` match the names that we configured for our source camera and verification system service, respectively.
+If you used different names, edit as applicable.
+View these [instructions to configure a transform camera](/build/configure/components/camera/transform/) for reference.
+Save your config.
+
+## View your verification system in action
+
+Navigate to the **Control** tab.
+Expand the card matching the name of the transform camera you configured to view an image stream with the system's notifications overlaid.
+With this configuration of detectors, your verification system will trigger a countdown when it detects people and disarm the alarm if it detects your face with the facial detector within the countdown period.
```

Comment:
This is the kind of info missing in the intro

Reply:
I moved this to the intro-- do you think that's ok? Seems weird to have the video in twice otherwise I would leave here as well